docs/content/cluster-scope/add_operator.md

```diff
@@ -0,0 +1,32 @@
+# Constructing a PR to add an Operator
+
+There are two methods one can follow to add an Operator to a cluster:
+1. Install via OLM
+1. Manual install
+
+## Install an operator via OLM
+
+Operator Lifecycle Manager (OLM) makes it easy to install and manage Operators on OpenShift clusters.
+To install an Operator via OLM, you must make a PR that does the following:
+
+* Create the `Subscription` to the Operator
+* Create the corresponding kustomization file
+* Pull the resource into the appropriate cluster overlay
+
+Your PR will add the necessary files for the Operator in the base folder and pull the resource in the desired overlays. An explanation of the folder structure can be found [here](folder-doc).
+
+In addition to the `Subscription`, you may also need to add a `Catalog source` or `Operator group`.
```

Comment:
How does the user know when/why to create a catalog source? 

When should they create an operator group?